src/tests/unit/asset.spec.ts

```diff
@@ -58,6 +58,23 @@ describe('Asset Class', () => {
       });
     });
 
+    it('should successfully call with valid data, Authorization and Network header', () => {
+      Configuration.setAuthTokens(accesToken, '');
+      pSdk.setNetwork('mainnet');
+      const assetDefaultsData = {
+        walletId: '6e081b82-dbed-4485-bdbc-a808ad911758',
```

Comment:
We've used this address (`6e081b82-dbed-4485-bdbc-a808ad911758`) 4 times in this PR - could it be a put into a variable?